docs/en/ingest-management/release-notes/release-notes-8.7.asciidoc

```diff
@@ -0,0 +1,222 @@
+// Use these for links to issue and pulls.
+:kibana-issue: https://github.com/elastic/kibana/issues/
+:kibana-pull: https://github.com/elastic/kibana/pull/
+:beats-issue: https://github.com/elastic/beats/issues/
+:beats-pull: https://github.com/elastic/beats/pull/
+:agent-libs-pull: https://github.com/elastic/elastic-agent-libs/pull/
+:agent-issue: https://github.com/elastic/elastic-agent/issues/
+:agent-pull: https://github.com/elastic/elastic-agent/pull/
+:fleet-server-issue: https://github.com/elastic/fleet-server/issues/
+:fleet-server-pull: https://github.com/elastic/fleet-server/pull/
+
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-8.7.0>>
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+// begin 8.7.0 relnotes
+
+[[release-notes-8.7.0]]
+== {fleet} and {agent} 8.7.0
+
+Review important information about the {fleet} and {agent} 8.7.0 release.
+
+[discrete]
+[[breaking-changes-8.7.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and
+performance. Before you upgrade, review the breaking changes, then mitigate the
+impact to your application.
+
+[discrete]
+[[breaking-147616]]
+.Remove the current_upgrades endpoint
+[%collapsible]
+====
+*Details* +
+The `api/fleet/current_upgrades` endpoint has been removed. For more information, refer to {kibana-pull}147616[#147616].
+*Impact* +
```

Comment:
```suggestion

*Impact* +
```

Reply:
You need a line break here. 
![image](https://user-images.githubusercontent.com/14206422/227580502-3f8dcf2b-8e2f-4c6a-9e72-b149251ddcf8.png)